executor/executor_test.go

```diff
@@ -2400,19 +2400,19 @@ func (s *testSerialSuite) TestSplitRegionTimeout(c *C) {
 	c.Assert(failpoint.Disable("github.com/pingcap/tidb/store/tikv/MockSplitRegionTimeout"), IsNil)
```

Comment:
Could we use `tikv.MockSplitRegionTimeout.Disable()` instead?

Reply:
done